### PR TITLE
Force README.rst to be read using utf-8 encoding

### PR DESCRIPTION
**Pull request by ericof**
This fixes issue #12

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 setup(name='pyroma',
       version=version,
       description="Test your project's packaging friendliness",
-      long_description=open("README.rst", 'rt').read() + "\n" +
+      long_description=open("README.rst", 'rt', encoding='utf-8').read() + "\n" +
```
Comment (4383) [inline setup.py]: Hi!

You should use a compatible python2|3 syntax like this:
```sh
long_description=open("README.rst", "rb").read().decode("UTF-8") + "\n" +
       open("HISTORY.txt", 'rt').read(),
       open("HISTORY.txt", 'rt').read()
```

**Comment by ericof** [comment]
@4383, thanks for the heads up

**Comment by 4383** [comment]
@ericof you are welcome!

**Comment by 4383** [comment]
@ericof You can squash your commits

**Comment by ericof** [comment]
@4383 Done

**Comment by 4383** [comment]
@ericof Thanks!
@regebro It's good for me, I thinks you can shipit.

**Comment by regebro** [comment]
Thanks for finding and fixing this bug. I did it somewhat differently, but it's solved now.

**Pull request closed without merging**